plugins/hls-cabal-plugin/src/Ide/Plugin/Cabal/Completion/CabalFields.hs

```diff
@@ -1,12 +1,16 @@
-module Ide.Plugin.Cabal.Completion.CabalFields (findStanzaForColumn, findFieldSection, getOptionalSectionName, getAnnotation, getFieldName, onelineSectionArgs) where
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+module Ide.Plugin.Cabal.Completion.CabalFields (findStanzaForColumn, findFieldSection, findTextWord, findFieldLine, getOptionalSectionName, getAnnotation, getFieldName, onelineSectionArgs, getFieldEndPosition, getSectionArgEndPosition, getNameEndPosition, getFieldLineEndPosition, getFieldLSPRange) where
```

Comment:
I am amazed stylish-haskell accepts this.

Would you mind introducing a couple of new lines to make this easier to read? E.g.

```suggestion
module Ide.Plugin.Cabal.Completion.CabalFields 
  ( findStanzaForColumn, 
    findFieldSection, 
    findTextWord, 
    ...
    ) where
```

or something like that?